kubel.el

```diff
@@ -190,7 +190,9 @@ CMD is the kubectl command as a list."
 
 CMD is the command string to run."
   (kubel--log-command "kubectl-command" cmd)
-  (shell-command-to-string cmd))
+  (with-output-to-string
+    (with-current-buffer standard-output
+      (shell-command cmd t "*kubel stderr*"))))
```

Comment:
Because it is only being used here, didn't think it was worth it to extract `*kubel stderr*` to a constant. Let me know if you think I should that.